exercises/reject_a_promise/problem.md

```diff
@@ -9,19 +9,8 @@ ever leaving either the resolved or rejected states.
 
 ## Task
 
-Use `q` again to create a promise.
-
 Create a function to print `error.message` using `console.log`.  Pass this
 function as a rejection handler to the `then` method of your promise.
 
 Manually reject that promise using `setTimeout` with a delay of 300ms and pass
 it an `Error` object with parameter `"REJECTED!"`;
```

Comment:
It might be good to call out the similarity to the previous exercise (and not use the word "manually").